src/sentry/scripts/tsdb/cmsketch.lua

```diff
@@ -373,26 +329,23 @@ function Command:new(fn, readonly)
     end
 
     return function (keys, arguments)
-        local defaults = nil
-        if not readonly then
```

Comment:
It seems like `readonly` is no longer used, should we remove it from the `Command:new` parameter list and get rid of the default-it-to-false check?